README.md

```diff
@@ -30,54 +30,48 @@
 mistakes when it comes to equity compensation.
 Talk to anyone who’s worked in startups and you’ll hear stories of how they or their
 colleagues met with the painful consequences of not learning enough up front.
 
-### Why this Guide?
+Over time, however, compensating employees with equity has become much more common not only in small new companies, but also in large mature companies. As of 2014, 7.2% of all private sector employees (~8.5M Americans) and 13.1% of *all* employees of companies with stock held stock options.Interestingly, this number has steadily declined since 2002 levels of 12.3% and 23.2% respectively ([NCEO analysis](https://www.nceo.org/assets/pdf/articles/GSS-2014-data.pdf) of General Social Survey data).
 
-Because you shouldn’t need a law degree or MBA to protect yourself when dealing with the
-details of equity compensation.
-Because you shouldn’t have to know someone kind and brilliant in financial ops or HR.
-Because you shouldn’t have to fail before you succeed.
+A company like Amazon or Google is decidedly less risky than a new company made up for a few people in a garage, so why do large companies offer equity?
 
-While [a lot of information](#further-reading) on this topic is just a Google search away,
-the best information is scattered about.
-Many blogs and articles focus only on a narrow topic, are getting older, or are on sites
-supported by ads or other products.
+First, ownership in a company is a good way to align employees interests with the company's interests. If everybody works hard and makes the company grow, the company and its employees all benefit.
 
-We’ll take you through the common scenarios, pitfalls, and best practices of negotiating
-equity compensation.
-An hour or two reading the material here and checking out the linked resources could
-ultimately be among the most financially valuable ways you spend your time.
+🌪 Second, and controversially, public companies offer executives equity compensation in order to make use of a tax loophole. In 1993,  President Bill Clinton attempted to limit executive pay with a new section of the Internal Revenue Code, 162(m). Unfortunately, the legislation backfired, as it created a loophole that made performance based pay—including stock options—fully tax deductibe, thereby creating a dramatic incentive to pay executives via stock options ([Balsam](https://www.epi.org/publication/taxes-executive-compensation/)).
 
-🚧 Groundwork:
-About how and why this was written.
+From 1970-79, the average compensation for a CEO of one of the 50 largest firms in the United States was $1.2M—11.2% of which was stock option-based. By 2000-05, the same numbers had risen to $9.2M and 37% respectively ([Frydman & Jenter](http://www.nber.org/papers/w16585.pdf), Fig. 2).
+
+* 🚧 Add concluding paragraph that ties everything together.
+* 🚧 The point about equity comp. aligning company and employee interests should be made sooner, but I'm totally fried right now. 
```

Comment:
Possible that the alignment point could be the opening salvo. Currently we're focusing on wealth at the very top thru the opening paragraphs--expectations of wealth is certainly an entry scenario! But I think the most important thing to open with is a response to the most common question we can imagine: What the eff is equity _for_? 

"Equity compensation is in its ideal form meant to align the interests of individual employees with the goals of a company. This alignment can have dramatic effects in creativity, team building, longevity of employment, and in the creation of wealth for the company as a whole and the individuals who work to make it a success. 

There are many other reasons companies from five-person startups to vast multilevel cooperations have built equity into their compensation packages over the last thirty years. Offering partial ownership in a company with the expectation of financial return is among the best methods to attract and retain the most talented people..."